lib/travis/build/bash/travis_install_jdk.bash

```diff
@@ -25,3 +41,18 @@ travis_install_jdk() {
   # shellcheck disable=2088
   travis_cmd "~/bin/install-jdk.sh --target \"$JAVA_HOME\" --workspace \"$TRAVIS_HOME/.cache/install-jdk\" --feature \"$version\" --license \"$license\"$certlink" --echo --assert
 }
+
+travis_install_jdk_package(){
+
+  local JAVA_VERSION
+  UNAME_ARCH=$(uname -m)
+  JAVA_VERSION="$1"
+  sudo apt-get update -yqq
+  PACKAGE="adoptopenjdk-${JAVA_VERSION}-hotspot"
+  if ! dpkg -s $PACKAGE >/dev/null 2>&1; then
+    if dpkg-query -l adoptopenjdk* >/dev/null 2>&1; then
+      dpkg-query -l adoptopenjdk* | grep adoptopenjdk | awk '{print $2}' | xargs sudo dpkg -P
+    fi
+    sudo apt-get -yqq --no-install-suggests --no-install-recommends install $PACKAGE && rc=$? || rc=$?
```

Comment:
@DamianSzymanski, maybe we should terminate in case of failure, as we do [here](https://github.com/travis-ci/travis-build/pull/1812/files#diff-0ee3eb26df64d72ab4cd5b53e2b620afR32)?